examples/gce-machinedeployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  # If you change the namespace/name, you must also
+  # adjust the rbac rules
+  name: machine-controller-gce
+  namespace: kube-system
+type: Opaque
+stringData:
+  serviceAccount: "<< GOOGLE_SERVICE_ACCOUNT >>"
+---
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineDeployment
+metadata:
+  name: gce-machinedeployment
+  namespace: kube-system
+spec:
+  paused: false
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  minReadySeconds: 0
+  selector:
+    matchLabels:
+      foo: bar
+  template:
+    metadata:
+      labels:
+        foo: bar
+    spec:
+      providerSpec:
+        value:
+          sshPublicKeys:
+            - "<< YOUR_PUBLIC_KEY >>"
+          cloudProvider: "gce"
+          cloudProviderSpec:
+            # If empty, can be set via GOOGLE_SERVICE_ACCOUNT env var
+            serviceAccount:
+              secretKeyRef:
+                namespace: kube-system
+                name: machine-controller-gce
+                key: serviceAccount
+            # See https://cloud.google.com/compute/docs/regions-zones/
+            zone: "europe-west3-a"
+            # See https://cloud.google.com/compute/docs/machine-types
+            machineType: "n1-standard-2"
+            # In GB
+            diskSize: 50
+            # Can be 'pd-standard' or 'pd-ssd'
+            diskType: "pd-standard"
+            labels:
+              "kubernetes_cluster": "my-cluster"
+          # Can be 'ubuntu', 'coreos' or 'centos'
```

Comment:
Not correct anymore. CentOS is not supported

Reply:
Gnah, got me.